recipes/github-activity/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "github-activity" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/github_activity-{{ version }}.tar.gz
+  sha256: 6749892ea6bff4c5e2b339269215883bc179a3c28eb6fdd57aa1ba43ca166269
+
+build:
+  entry_points:
+    - github-activity = github_activity.cli:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - git
+    - markdown
+    - numpy
+    - pandas
+    - python >=3.6
+    - python-dateutil
+    - requests
+    - tqdm
+
+test:
+  imports:
+    - github_activity
+  commands:
+    - pip check
+    - github-activity --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/executablebooks/github-activity
+  summary: Simple markdown changelogs for GitHub repositories
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
```

```suggestion
    - jtpio
    - manics
```
I'm happy to be added!

---

Thanks!